### PR TITLE
Land blocks - Diag Tools

### PR DESCRIPTION
**Pull request by Forbiddenz**
Type Diag from the console to start landblock diagnostics.
Start of loading, unloading landblocks.

### CODE REVIEW

File: Source/ACE.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
```
Comment (Mogwai-AC) [inline Source/ACE.sln]: Yeah, no-go.

File: Source/ACE.sln [hunk 2]
```diff
@@ -45,44 +45,86 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
...
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
 		Debug|x64 = Debug|x64
+		Release|Any CPU = Release|Any CPU
 		Release|x64 = Release|x64
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{1CE9FC39-B920-4120-939D-1C0AA6FF017C}.Debug|Any CPU.ActiveCfg = Debug|x64
```
Comment (Mogwai-AC) [inline Source/ACE.sln]: You need to remove all non-x64 configurations.

File: Source/ACE/Managers/LandblockManager.cs [hunk 3]
```diff
@@ -86,7 +85,9 @@ private static Landblock GetLandblock(LandblockId landblockId, bool propogate)
                     if (landblocks[x, y] == null)
                     {
                         // load up this landblock
-                        Landblock block = new Landblock(landblockId);
+                        Landblock block = new Landblock(landblockId, x, y);
```
Comment (Mogwai-AC) [inline Source/ACE/Managers/LandblockManager.cs]: x and y are completely redundant with landblockId.

File: Source/ACE/Entity/Landblock.cs [hunk 4]
```diff
@@ -50,16 +54,24 @@ internal class Landblock
...
 
         public LandblockId Id
         {
             get { return id; }
         }
 
-        public Landblock(LandblockId id)
+        public Landblock(LandblockId id, int r, int c)
         {
             this.id = id;
```
Comment (Mogwai-AC) [inline Source/ACE/Entity/Landblock.cs]: r and c are redundant with landblockId.

File: Source/ACE/Managers/LandblockManager.cs (hunk 3 above)
Comment (Forbiddenz) [inline Source/ACE/Managers/LandblockManager.cs]: How do I get x, and y from landblock ?
Keep in mind I am trying to draw this kn a grid of 256x256 in pixels

Comment (Mogwai-AC) [inline Source/ACE/Managers/LandblockManager.cs]: LandblockId is a struct. It has properties for LandblockX and LandblockY.

**Pull request closed without merging**